src/test/java/com/ericsson/ei/flowtests/FlowTestExternalComposition.java

```diff
@@ -16,55 +16,50 @@
 @SpringBootTest
 public class FlowTestExternalComposition extends FlowTestBase {
 
-     private static Logger log = LoggerFactory.getLogger(FlowTest.class);
-     static protected String inputFilePath = "src/test/resources/aggregatedExternalComposition.json";
+    private static final String RULES_FILE_PATH = "src/test/resources/ArtifactRules_new.json";
+    private static final String EVENTS_FILE_PATH = "src/test/resources/test_events.json";
+    private static final String AGGREGATED_OBJECT_FILE_PATH = "src/test/resources/aggregatedExternalComposition.json";
+    private static final String AGGREGATED_OBJECT_ID_1 = "6acc3c87-75e0-4b6d-88f5-b1a5d4e62b43";
+    private static final String AGGREGATED_OBJECT_ID_2 = "cfce572b-c3j4-441e-abc9-b62f48080ca2";
+    private static final String AGGREGATED_OBJECT_ID_3 = "cfce572b-c3j4-441e-abc9-b62f48080ca2";
+
+    @Override
+    String setRulesFilePath() {
+        return RULES_FILE_PATH;
+    }
 
-        protected ArrayList<String> getEventNamesToSend() {
-             ArrayList<String> eventNames = new ArrayList<>();
-             eventNames.add("event_EiffelArtifactCreatedEvent_3");
-             //eventNames.add("event_EiffelCompositionDefinedEvent_4");
-             eventNames.add("event_EiffelArtifactCreatedEvent_4");
-             //eventNames.add("event_EiffelCompositionDefinedEvent_5");
-             eventNames.add("event_EiffelArtifactCreatedEvent_5");
-             return eventNames;
-        }
+    @Override
+    String setEventsFilePath() {
+        return EVENTS_FILE_PATH;
+    }
 
-        protected void checkResult() {
-            try {
-                String expectedDocuments = FileUtils.readFileToString(new File(inputFilePath), "UTF-8");
-                ObjectMapper objectmapper = new ObjectMapper();
-                JsonNode expectedJsons = objectmapper.readTree(expectedDocuments);
-                JsonNode expectedJson1 = expectedJsons.get(0);
-                JsonNode expectedJson2 = expectedJsons.get(1);
-                JsonNode expectedJson3 = expectedJsons.get(2);
-                String document1 = objectHandler.findObjectById("6acc3c87-75e0-4b6d-88f5-b1a5d4e62b43");
-                String document2 = objectHandler.findObjectById("cfce572b-c3j4-441e-abc9-b62f48080ca2");
-                String document3 = objectHandler.findObjectById("cfre572b-c3j4-4d1e-ajc9-b62f45080ca2");
-                JsonNode actualJson1 = objectmapper.readTree(document1);
-                JsonNode actualJson2 = objectmapper.readTree(document2);
-                JsonNode actualJson3 = objectmapper.readTree(document3);
-                int i = 0;
-            } catch (Exception e) {
-                log.info(e.getMessage(),e);
-            }
+    @Override
+    List<String> setEventNamesToSend() {
+        List<String> eventNames = new ArrayList<>();
+        eventNames.add("event_EiffelArtifactCreatedEvent_3");
+        //eventNames.add("event_EiffelCompositionDefinedEvent_4");
```

Comment:
Will these event not be used?

Reply:
This test was not workable.. and part of events were commented. so I left them for future

Reply:
Ok,, checked the git log and those were commented out since earlier. So leave it as is.